### DISCUSSION TITLE
dimming HmIP-BDT with alexa-smart-home-v3

### DISCUSSION THREAD

**Discussion by HMside** [rdmtc/RedMatic#590] [Q&A]
Hello,

do any of you have an example flow to control a HmIP-BDT dimmer via alexa-smart-home-v3?

I am currently dividing the incoming payload by 100 with the calculator to send the correct value (LEVEL) to the dimmer.

That works so far, but Alexa always complains and says "Living room light not responding...".

**Comment by hobbyquaker** [comment]
Support-Frage zu einem Fremd-Node; mit RedMatic 9 (https://github.com/rdmtc/RedMatic/releases) bitte erneut versuchen. Geschlossen.

---
*Written by Claude Fable on behalf of hobbyquaker.*
